### PR TITLE
[SVLS-4918] workflow to update node extension versions

### PR DESCRIPTION
**Pull request by apiarian-datadog**
Adapted from [the datadog-aas-extension version](https://github.com/DataDog/datadog-aas-extension/blob/master/.github/workflows/node_extension_update_versions.yml)

### CODE REVIEW

**Comment by apiarian-datadog** [comment]
https://github.com/DataDog/datadog-aas-linux/pull/50 worked for PHP
https://github.com/DataDog/datadog-aas-linux/pull/47 worked for .NET
https://github.com/DataDog/datadog-aas-linux/pull/48 worked for python
https://github.com/DataDog/datadog-aas-linux/pull/49 worked for java
https://github.com/DataDog/datadog-aas-linux/pull/46 worked for Node

File: datadog_wrapper [hunk 1]
```diff
@@ -262,7 +262,7 @@ setUpPyEnv() {
         DD_PYTHON_TRACER_VERSION=2.7.4
     fi
 
-    pip install ddtrace
+    pip install "ddtrace==${DD_PYTHON_TRACER_VERSION}"
```
Comment (apiarian-datadog) [inline datadog_wrapper:265]: looks like we weren't pinning this before.

**Comment by apiarian-datadog** [comment]
new pull requests generated by python scripts:
.NET: https://github.com/DataDog/datadog-aas-linux/pull/54
python: https://github.com/DataDog/datadog-aas-linux/pull/53
java: https://github.com/DataDog/datadog-aas-linux/pull/52
php: https://github.com/DataDog/datadog-aas-linux/pull/51
node for tracer v4: https://github.com/DataDog/datadog-aas-linux/pull/55
(v5 doesn't need an update yet, but it's fine)

**Review by jcstorms1 — APPROVED** [review]
LGTM 👍

**Pull request merged**